README.md

```diff
@@ -1 +1,213 @@
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[diplay score on UI]**
+- **[using javascript to render dynamically list of scores]**
+- **[using webpack bundle files of project]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- [Live Demo Link](coming soon ....)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A browser of you choice.
+- A text editor of your choice.
+- An installed node.js on your local system
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-folder
+  git clone https://github.com/clin2on3mun/LeaderBoard.git
+
+
+### Install
+
+Install this project with:
+
+-you can download it as a zipped file
+-you can clone it using OS terminal or code editor terminal-
+-you should use npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+-npm run start in terminal and choose the browser  
+
+### Deployment
+
+You can deploy this project using:
+
+You can deploy this project using:
+- Free deployment services like GitHub pages.
+- Any deployment services of your choice.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Munana Clinton**
+
+- GitHub: [@githubhandle](https://github.com/clin2on3mun)
+- Twitter: [@twitterhandle](https://twitter.com/ClintonMunana)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/munana-clinton-b6122720b/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **[style the UI to look better]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can recommend to your friend, family and colleagues who want to use my services
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+**I would like to thank microverse for guiding and providing with all great material to do my projects and thank my coding partners who helped**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
-  [ ] Your MIT license link currently leads to an error page. I have attached a screenshot below. I think this might be because you don't have a license file in your project. So I will suggest that you include an MIT license file in your project. You can use this  [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to learn how to add one to your project :wink:

<img width="768" alt="Screen Shot 2023-05-16 at 12 07 36 PM" src="https://github.com/clin2on3mun/LeaderBoard/assets/102699958/b6fbb62f-b961-486e-8846-bc2fc5a49853">